### PR TITLE
Update copyright info

### PR DESCRIPTION
**Pull request by Umcaruje**
Addresses issue #1539

### CODE REVIEW

**Comment by Spekular** [comment]
What should we do with this pr? Merge, close, or tag Toby and ask him?

File: plugins/LadspaEffect/swh/gsm/add.c [hunk 1]
```diff
@@ -125,7 +125,7 @@ word gsm_norm P1((a), longword a )
  *
  *
  * and for negative values on the interval with
- * minimum of -2147483648 (-10000000000000000000000000000000) and
+ * minimum of -2015483648 (-10000000000000000000000000000000) and
```
Comment (tresf) [inline plugins/LadspaEffect/swh/gsm/add.c]: BAD

**Comment by tresf** [comment]
Man that is a lot of files.

Well, I just found at least one mistake.  If someone has the time to spot check all 600+ files, I'd be happy to merge it, but GitHub won't display them all and I don't have the time to do the diff here locally and compare.

**Comment by tresf** [comment]
Also, we don't want to traverse into any 3rd party code at all since it could differ from upstream.  We want to get as much 3rd party code from upstream as possible and when we sync the code in, we don't want to get hits on copyright dates.  This means we need to know the difference between our own plugins and 3rd party ones (which some of us can name off of the top of our head). 

I vote we close this as I see virtually no value given the amount of time to do it properly.

**Comment by Umcaruje** [comment]
>I vote we close this as I see virtually no value given the amount of time to do it properly.

I agree.

**Comment by Wallacoloo** [comment]
```
$ lmms --version
LMMS 1.1.0
(Linux x86_64, Qt 4.8.6, GCC 4.9.1)

Copyright (c) 2004-2014 LMMS developers.
...
```

Is it worth at least fixing the externally visible copyright dates? I think those are just the dates from the "About" dialog box and the above `--version` and `--help` flags. These could be made to make use of the build date to prevent having to do the same thing come 2016.

**Pull request closed without merging**